src/main/kotlin/com/demonwav/mcdev/platform/mcp/McpModuleType.kt

```diff
@@ -33,4 +34,6 @@ object McpModuleType : AbstractModuleType<McpModule>("", "") {
     override val hasIcon = false
 
     override fun generateModule(facet: MinecraftFacet) = McpModule(facet)
+
+    val MC_1_12_2 = SemanticVersion.release(1, 12, 2)
```

Comment:
We should probably create a "Minecraft versions" `object` or something to hold all of the random MC versions we have laying around.